app/webapp/src/main/webapp/assets/css/annotare.css

```diff
@@ -118,7 +118,12 @@
     font-weight: lighter;
 }
 
-#main-content-area section {
+section#main-content-area {
```

Comment:
Just curious if any specific reason to change this? I assume the previous one would apply style to all `section` tags inside `main-content-area`. However, this current one would just be applied to `section` tag with `main-content-area` id.

Reply:
Previous one applied padding to all child `sections` tags of element with id `main-content-area`. But I wanted to add padding to only that particular section with that id.